features/steps/test_starter/{{ cookiecutter.repo_name }}/ipython_script.py

```diff
@@ -0,0 +1,10 @@
+if __name__ == "__main__":
+    from IPython.testing.globalipapp import get_ipython
+    print("!!!****!!!****")
```

Comment:
Is this needed, or just for testing it worked?

Reply:
It's not needed